sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushDownUtils.scala

```diff
@@ -116,7 +116,11 @@ object PushDownUtils extends PredicateHelper {
   }
 
   /**
-   * Pushes down LIMIT to the data source Scan
+   * Pushes down LIMIT to the data source Scan.
+   *
+   * @return the tuple of Boolean. The first Boolean value represents whether to push down, and
+   *         the second Boolean value represents whether to push down partially which means to
+   *         keep the `Limit`.
```

Comment:
can we be a little more clear and say something like 
`...the second Boolean value represents whether to push down partially, which means Spark will keep the Limit  and do it again.`

Reply:
OK